### ISSUE TITLE
W and S navigation disabled after jumping with JW or JS

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#386]
While testing version [dev 21](https://www.colorado.edu/physics/phet/dev/html/balloons-and-static-electricity/1.4.0-dev.21/phet/balloons-and-static-electricity_all_phet_debug.html?accessibility) on April 5th, 2018.

I can replicate issue for SW navigation keys not working after jumping

- Using the JW key seems to disable the W key on the first step after a JW jump.
- Using the JS keys seems to disable the S key on the first step after a JS jump.

The JC and JN jump keys do not seem to be affected. 
This happens in Safari whether the screen reader is on or off, and in Firefox and Chrome (only tested without screen reader).

It must seems it could have something to do with the W and S keys specifically, since a JW disables W and a JS disables S.

**Comment by terracoda** [comment]
Good news, it that it is easy to get things working again, just press an Arrow key or any other of the WASD keys.

**Comment by jessegreenberg** [comment]
Thanks  @terracoda, this really helps isolate the problem.

**Comment by phet-steele** [comment]
@terracoda is this a duplicate of #365? If so, please close one of these two issues.

**Comment by phet-steele** [comment]
Thanks @terracoda, I noticed you closed #365.

This issue is for phetsims/QA/issues/112.

**Comment by jessegreenberg** [comment]
@terracoda @phet-steele I think this is fixed in master due to changes made for #389. Could you please verify on phettest?

**Comment by terracoda** [comment]
@jessegreenberg, I will verify once phettest will load for me. Every now and again, it stops loading.

**Comment by terracoda** [comment]
@jessegreenberg, the jump keys are working fine now. I will close this issue. Thanks!

**Comment by phet-steele** [comment]
@jessegreenberg I'm experiencing new funky-ness. Stand by.

**Comment by terracoda** [comment]
Sorry @phet-steele, I won't close anymore issues, you are testing too on different platforms.

**Comment by jessegreenberg** [comment]
@phet-steele showed me that the funkiness is that it is not possible to  hold the "J" key and press "W" and "S" rapidly to move the balloon back and forth between the sweater and wall. Presumably because the interval introduced to the KeyboardDragListener for hotkeys is stopping multiple presses.

**Comment by jessegreenberg** [comment]
Thanks very much for finding this @phet-steele! I think this should be fixed with the above, can you please verify?

**Comment by terracoda** [comment]
@phet-steele, yes thanks! I never even would have thought to try to do that ;-)

**Comment by phet-steele** [comment]
@jessegreenberg this is working great for me now, thanks.
